lib/engine/game/g_1822.rb

```diff
@@ -33,19 +33,35 @@ class G1822 < Base
       GAME_INFO_URL = 'https://google.com'
 
       HOME_TOKEN_TIMING = :operating_round
+      MUST_BUY_TRAIN = :always
+      NEXT_SR_PLAYER_ORDER = :most_cash
 
-      BIDDING_TOKES = {
+      BIDDING_TOKENS = {
         "3": 6,
         "4": 5,
         "5": 4,
         "6": 3,
         "7": 3,
       }.freeze
 
+      BIDDING_TOKENS_PER_ACTION = 3
+
       include StubsAreRestricted
 
-      def setup
-        setup_minors
+      attr_accessor :bidding_token_per_player
+
+      def entity_can_use_company?(_entity, company)
+        # Setting bidding companies owner to bank, make sure the abilities dont show for theese
+        company.owner != @bank
+      end
+
+      def init_round
+        stock_round
+      end
+
+      def operating_order
```

Comment:
maybe we should separate this and sorted_corporations out into an include since it's the same for 1867 and 1822

Reply:
That would be a good idea, i found more code that we can share in a include. i e with 1817 i have copied:

`      def format_currency(val)
        # On dividends per share can be a float
        # But don't show decimal points on all
        return format('$%.1<val>f', val: val) if val.is_a?(Float) && (val % 1 != 0)

        self.class::CURRENCY_FORMAT_STR % val
      end`

I can wait to extract them into an include until we see how much more i need from any specific title that isnt already in an include.